.github/workflows/cypress-e2e-parallelization.yml

```
@@ -34,6 +34,15 @@ on:
         required: true
       registry_password:
         required: true
+      CLIENT_ID :
+        required: true
+      CLIENT_SECRET:
+        required: true
+      JIRA_USER: 
+        required: true
+      JIRA_TOKEN_TEST: 
+        required: true
```

to keep consistency with secrets in reusable workflows
```suggestion
      client_id :
        required: true
      client_secret:
        required: true
      jira_user: 
        required: true
      jira_token_test: 
        required: true
```